src/r2mm/installing/profile_installers/ModLoaderVariantRecord.ts

```diff
@@ -1,217 +1,61 @@
 import ModLoaderPackageMapping from '../../../model/installing/ModLoaderPackageMapping';
-import { PackageLoader } from '../../../model/installing/PackageLoader';
+import {
+    GetInstallerVariantFromName,
+    PackageLoader,
+} from '../../../model/installing/PackageLoader';
 import VersionNumber from '../../../model/VersionNumber';
+import { EcosystemSchema } from '../../../model/schema/ThunderstoreSchema';
 
+export type GAME_NAME = string;
```

Comment:
IMO as-is, this adds no value but might confuse others to assume GAME_NAME is actually based on some hard coded allow list. If this is still needed for the same function the old, stronger typing had, it should have the same typing as previously. If that function no longer makes sense after the other changes, we should drop the the custom type and use "string" instead.